README.md

```diff
@@ -18,20 +18,18 @@ This is how the SongDrive Dashboard currently looks like.
 
 ## Features
 
-- Create, edit and delete songs
-- Normal and viewport optimized fullscreen view for a single song
-- Switch song languages
-- Filter songs and setlists on typing, by tags and by key
-- Transpose song key
-- Create, edit and delete setlists
-- Rearrange songs in setlists with simple drag and drop
-- Presentation mode for setlists with device synchronization
-- PDF export for songs, setlists and songsheets
-- Plain text, SNG and Slack formatted export for songs and setlists
-- Hotkey support
-- General statistics, setlist and user specific stats
+- Create and edit songtexts with chords using a [simple syntax](https://github.com/devmount/SongDrive/blob/main/src/docs/docs.en.md#song-syntax)
+- Show single songs with a lot of tools or in a viewport optimized fullscreen view
+- Maintain song translations and wwitch song languages with a single click
+- Live-search through songs and setlists or filter by tags and by key
+- Live-transpose song key or save transposed songs on setlists
+- Create setlists and rearrange songs with simple drag and drop
+- Present setlists for on-stage performance with device synchronization
+- Export songssheets and setlists as plain text, PDF, SNG or Slack formatted
+- Use Hotkeys for almost anything (hotkey cheatsheet included)
+- Analyze general, setlist and user specific stats
 - UI supports multiple languages (currently: EN, DE)
-- Export and Import of complete SongDrive data
+- Many admin functions including export and Import of complete SongDrive data and managing users with [different roles](https://github.com/devmount/SongDrive/blob/main/src/docs/docs.en.md#user-roles)
```

Comment:
This is great. Can you include the user account too?
https://github.com/devmount/SongDrive/blob/main/src/docs/docs.en.md#account

Reply:
Done ✅